tests/test_mutations.py

```diff
@@ -412,6 +412,39 @@ def test_EL2(self):
                         f"summation on row {ii} does not equal alpha ratio"
                     )
 
+    def test_MicrosatMutationModel(self):
+        lo = 1
+        hi = 10
+        m = 1.0
+        u = 0.62
+        v = 0.015
+        p = 0
+        s = 0.88
+        model = msprime.MicrosatMutationModel(s, u, v, p, m, lo, hi)
+        row_sums = model.transition_matrix.sum(axis=1)
+        d = row_sums - model.transition_matrix.diagonal()
+        betas = [1 + ((i - lo) * s) for i in range(lo, hi + 1)]
+        alphas = [u - (v * (i - lo)) for i in range(lo, hi + 1)]
+        for j in range(len(betas)):
+            assert (d[0] / d[j]) - (
+                betas[0] / betas[j] * alphas[0] / (1 - alphas[j])
+            ) < 1e-6
+        # 2nd parameterization, slight variation
```

Comment:
Glad you're doing multiple parameterizations - perhaps make sure to set `p` not equal to zero in one of them? And it'd be nicer to do this with `@pytest.mark.parametrize`?

Reply:
will do